src/content/docs/design/customize-with-code/styling-with-css.mdx

```diff
@@ -18,6 +18,12 @@ For example, you can:
 - Set the primary button’s background color with `--kinde-button-primary-background-color`.
 - Set the color of a form field’s invalid message with `--kinde-control-associated-text-invalid-message-color` (or use `--kinde-shared-color-invalid` for a more global approach).
 
+## Storing your CSS and other asset files
```

Comment:
@clairekinde11 I would use "Hosting external CSS and other assets" - I think I wrote "storing" somewhere but it doesnt really make sense as it's not like Google Drive / Dropbox where you are storing files, it's more of a hosting scenario where they are made available for use in your code